src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -492,6 +492,9 @@ protected virtual void ShutDownCore(bool shutdownHost)
             Dispose();
             PreferenceSettings.SaveInternal(pathManager.PreferenceFilePath);
 
+            //Remove reference on shutdown
+            PreferenceSettings.dynamoModelRuntimePreferenceSettings = null;
```

Comment:
Not sure if we need to do this but may be good due to not being able to properly discuss the `DynamoModel` on shutdown.

Reply:
discuss == dispose?

Reply:
I think this is likely safe on DynamoModel if you ensure that `dynamoModelRuntimePreferenceSettings` is always reset to something valid on startup.

Can you add a test for that?